apps/accounts/views/login.py

```diff
@@ -20,24 +16,6 @@ class LoginView(LoginView):
     def dispatch(self, request, *args, **kwargs):
         return super().dispatch(request, *args, **kwargs)
 
-    def form_valid(self, form):
```

Comment:
This can all be removed because it overrides a parent method to do only password expiration checks.